corehq/apps/hqmedia/models.py

```diff
@@ -548,21 +543,6 @@ def get_form_name(self, lang=None):
         return self._get_name(self.form_name, lang=lang)
 
 
-def _log_media_deletion(app, deleted_media):
```

Comment:
This one also has to do https://dimagi-dev.atlassian.net/browse/ICDS-2 which is still open, but are you finding it useful? Every time I've spot-checked this assert, the deleted files have been genuinely no longer used in the app. But since the media doesn't get deleted, the alert continues to go off and contains an ever-increasing list of files that's difficult to parse.

Reply:
we decided to close that ticket since this is not happening anymore. Following up on that.